test/camunda-cloud/ElementTemplatesValidatorSpec.js

```diff
@@ -0,0 +1,24 @@
+import { CloudElementTemplatesValidator } from 'lib/camunda-cloud/ElementTemplatesValidator';
+
+
+describe('camunda-cloud - ElementTemplatesValidator', function() {
+
+
```

Comment:
Minor code style suggestion: only use 1 new line at the start
```suggestion

```

Reply:
👍

Reply:
Looked weird indeed.